javascript/packages/linter/docs/rules/erb-prefer-image-tag-helper.md

```diff
@@ -0,0 +1,47 @@
+# Linter Rule: Prefer `image_tag` helper over manual `<img>` tags
+
+**Rule:** `erb-prefer-image-tag-helper`
+
+## Description
+
+Prefer using Rails' `image_tag` helper over manual `<img>` tags with `image_path` or `asset_path`.
+
+## Rationale
+
+- The `image_tag` helper properly escapes the `src` value
+- Reduces template complexity
+- Simplifies adding attributes
+- Ensures consistent rendering
+- Prevents interpolation issues
+
+## Examples
+
+### ✅ Good
+
+```erb
+<%= image_tag "logo.png", alt: "Logo" %>
+<%= image_tag "banner.jpg", alt: "Banner", class: "hero-image" %>
+<%= image_tag "icon.svg", alt: "Icon", size: "24x24" %>
+
+<!-- Static image paths are fine -->
+<img src="/static/logo.png" alt="Logo">
+
+<!-- Dynamic URLs without helpers are fine -->
+<img src="<%= user.avatar.url %>" alt="User avatar">
```

Comment:
I wonder if this should also be flagged as an offense and that something like should be preferred:

```erb
<%= image_tag user.avatar.url, alt: "User avatar" %>
```

Reply:
Updated in [51b96ff](https://github.com/marcoroth/herb/pull/307/commits/51b96ff636b90edd63ce0b646cf00abde6d4b6cc)